src/test/java/com/github/fishio/audio/AudioTestUtil.java

```diff
@@ -36,7 +36,7 @@ public static boolean supportsMusic() {
 	public static boolean supportsSoundEffects() {
 		try {
 			Path path = AudioUtil.getAudioFiles(true).get(0);
-			new SoundEffect(path.toUri().toString());
+			new SoundEffect(path.toUri().toString()).isPlaying();
```

Comment:
@MrCrim This is actually something that fixes a bug where it is not properly checked. It has to do with delayed initialization, where simply creating the object does not actually test the audio drivers. Calling isPlaying forces java to initialize the object, so it then performs the check.

This actually fixes the bug of this not working. The reason there were no problems with travis is that it apparently supports one of the two ways we play sounds.

<bug